internal/provider/meshzoneegress_resource_plan_modify.go

```diff
@@ -0,0 +1,87 @@
+package provider
```

Comment:
this already should've been there @harshadixit12 - do you know what happened here? did you run `make` before the previous release?

Reply:
Looks like I bumped the version [here](https://github.com/Kong/terraform-provider-konnect-beta/pull/144/commits/4920eb15cf99a8e0c07f8ccaa143652442e5c20c) and ran `make speakeasy` afterwards, but not `make generate`.